puppet/deps.yaml

```diff
@@ -1,2 +1,2 @@
-puppetlabs-stdlib: 7.1.0
-puppetlabs-concat: 7.0.1
+puppetlabs-stdlib: 8.1.0
+puppetlabs-concat: 7.1.1
```

Comment:
I’m reminded here of my unhappiness that `puppet module install` doesn’t check any hashes. But that’s probably difficult.